website/docs/podman/rosetta.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 20
+title: Native Apple Rosetta translation layer (macOS)
+description: Use Apple Rosetta to speed up cross-architecture containers
+---
+
+# Native Apple Rosetta translation layer
+
+Virtual machine's created by Podman Machine use the native Apple hypervisor `applehv` with Rosetta enabled by default. This increases the speed of any `x86_64` builds or containers to near-native levels by using a translation layer.
+
+For more information, refer to the [official Apple documentation](https://developer.apple.com/documentation/virtualization/running_intel_binaries_in_linux_vms_with_rosetta) regarding the Rosetta technology.
+
+Rosetta support is **enabled by default** on all new Podman Machine installations. If you disable Rosetta, [qemu](https://www.qemu.org/) will instead be used.
+
+#### Prerequisites
+
+- macOS Silicon
+
+#### Procedure
+
+To enable Rosetta support, your Podman Machine instance must be re-created.
+
+1. Delete your Podman Machine.
+
+2. Enable Rosetta support under **Settings**:
+
+![rosetta](img/rosetta.png)
+
+3. Re-create your Podman Machine.
+
+#### Verification
+
+To verify that Rosetta has been enabled, check your `~/.config/containers/containers.conf` configuration.
+
+If `rosetta = false` does _not_ exist, or is missing, Rosetta is already enabled by default.
```

Comment:
does not exist or is missing sounds the same thing. What does it actually depict?
In the procedure part above, we asked users to enable the Rosetta support and here, we are saying `Rosetta is already enabled by default.` --- contradictory statements.

Should we simply say:
Open your `containers.conf` file
Check that the value of the `rosetta` parameter is set to `true`.

Reply:
rosetta parameter is true by default, but does not show in containers.conf.

The doc is more intended for users who haven't upgraded past 5.0.0 yet, or with to disable / enable rosetta support.